src/uiSelectController.js

```diff
@@ -163,8 +161,12 @@ uis.controller('uiSelectCtrl',
   };
 
   ctrl.focusSearchInput = function (initSearchValue) {
-    ctrl.search = initSearchValue || ctrl.search;
+    if (initSearchValue) {
+      ctrl.search = initSearchValue.val() || ctrl.search;
```

Comment:
I don't agree that `initSearchValue` should be a DOM element, angular controllers should avoid dealing with DOM. Can you look at leaving this as text string?